### PR TITLE
Add support and docs for tick tuple support

### LINKED ISSUE #116: Handle tick tuples in Bolt base class
Streamparse's support of the Storm's tick tuple isn't obvious. Adding something like this
```python
    def is_tick_tuple(self, tup):
        return tup.component == '__system' and tup.stream == '__tick'
```
to class streamparse.bolt.Bolt could help newbies (like me).

Also modifying wordcount.clj will be helpful too:
```closure
"count-bolt" (python-bolt-spec
...
:p 2
:conf {"topology.tick.tuple.freq.secs", 60}
)
```

### PR DESCRIPTION
**Pull request by cabiad**
Closes #116

### CODE REVIEW

File: doc/source/quickstart.rst [hunk 1]
```diff
@@ -381,6 +381,43 @@ adding class variables set to false for: ``auto_ack``, ``auto_anchor`` or
...
+        def process_tick(self, freq):
+            # An action we want to perform at some regular interval...
+            self.flush_old_state()
+
+Then, for example, to cause ``process_tick()`` to be called every
+2 seconds on all of your bolts that override it, you can launch
+your topology under ``sparse run`` by setting the appropriate -o
+option and value as in the following example:
+
+.. code-block:: bash
```
Comment (cabiad) [inline doc/source/quickstart.rst]: I'm not actually sure whether this will work.

File: doc/source/api.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ Bolts
 -----
 
 .. autoclass:: streamparse.bolt.Bolt
-    :members: ack, emit, emit_many, fail, initialize, process, log, auto_anchor, auto_ack, auto_fail
+    :members: ack, emit, emit_many, fail, initialize, process, process_tick, log, auto_anchor, auto_ack, auto_fail
```
Comment (dan-blanchard) [inline doc/source/api.rst]: Not that this is @cabiad's fault, but why aren't we just using `:members:` by itself here, which will automatically update?

**Comment by dan-blanchard** [comment]
The build here failed because of a stupid sporadic timing issue with our tests on Travis.  When Travis is swamped, the Docker containers run slower, so sometimes the sleep times we have in our tests aren't long enough.  I haven't come up with a good work around yet (other than just constantly bumping up the sleep times when I encounter this).

**Comment by dan-blanchard** [comment]
This looks good to me.  Thanks for the contribution! :+1:

**Pull request merged**